django/locale/translation/translations.json

```diff
@@ -3218,5 +3218,9 @@
     "Maximum sources": {
         "fr": "",
         "fr_auto": "Sources maximales"
+    },
+    "Otto": {
+        "fr": "Otto",
+        "fr_auto": "Otton"
```

Comment:
French auto has "n"

Reply:
Gets overriden by "fr" value (manually added to address this problem).